config/config.go

```diff
@@ -160,9 +167,20 @@ func (c *S3Cli) allowMultipart() bool {
 			return false
 		}
 	}
+	if c.isAlicloudHost() {
```

Comment:
I think we're missing tests for multipart alicloud, deleting/commenting this block of code doesn't cause any failures.

Reply:
HI @luan Same as gcp, the multipart of s3 manager does not support alicloud.